packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -294,6 +298,7 @@ const start = async (zcf, privateArgs) => {
       makeCreateQuote({ overridePrice: median, timestamp }),
     );
 
+    // if (TimeMath.compareAbs(TimeMath.coerceTimestampRecord(timestamp), publishedTimestamp) === -1) {
```

Comment:
If this isn't changed to the commented out version, does anything verify that the brands are the same? Why not use the commented-out version?

Reply:
the current branch does:
```js
    // If a more recent timestamp has already been published, we are too late.
    if (TimeMath.compareAbs(timestamp, publishedTimestamp) < 0) {
      return;
    }
```

but `compareAbs` does not compare brands. The previous `if (timestamp < publishedTimestamp)` didn't either, so the only improvement here is to correctly handle our TimerService emitting branded `timestamp` records.

At this point, we're only doing a partial conversion: vat-timer will emit brands, but will tolerate both branded and unbranded inputs. I think when we make that later step, and all timestamps must be branded everywhere, then we should change `compareAbs` to mandate the two sides are both branded.

But, it *would* make a lot of sense to change `compareAbs()` to use the internal `sharedTimerBrand()` to assert that *if* both sides have a brand, then they must match. I'll try that and see if anything breaks.